docs/release.rst

```diff
@@ -36,7 +38,8 @@ Enhancements
 
 * Adds the MongoDB-backed :class:`zarr.storage.MongoDBStore` class enabling a
   MongoDB database to be used as the backing store for an array or group.
-  By :user:`Joe Hamman <jhamman>`, :issue:`299`, :issue:`372`.
+  By :user:`Joe Hamman <jhamman>`, :user:`John Kirkham <jakirkham>`,
+  :issue:`299`, :issue:`372`, :issue:`401`.
```

Comment:
can we add @nbren12 to the list of contributors here? He prototyped this and though his work doesn't show up in the commit record, he deserves 95% of the credit.

Reply:
+1 to adding @nbren12, I think we should credit all people who contributed towards it.

Reply:
Yep, agreed. Sorry this was an oversight.